stream/src/test/java/org/apache/kafka/test/MockProcessorContext.java

```diff
@@ -147,11 +157,18 @@ public long timestamp() {
         return this.timestamp;
     }
 
-    public void node(ProcessorNode node) {
+    public void pushNode(ProcessorNode node) {
```

Comment:
MockProcessorContext have the same logic for a bunch of the functions as ProcessorContextImpl, so share we just make it implement the interface while extending ProcessorContextImpl and just override other functions?